src/main/java/edu/cornell/kfs/module/purap/jaggaer/contract/xml/Attachment.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.module.purap.jaggaer.contract.xml;
+
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import jakarta.xml.bind.annotation.XmlType;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {
+    "attachmentFile"
+})
+@XmlRootElement(name = "Attachment")
+public class Attachment {
```

Comment:
There is an Attachment object in Jaggaer supplier, the two are very different.  I wonder if the two should be renamed so they are context specific?  Like this one being AttachmentContract, and the other being AttachmentSupplier?

Reply:
I'm wondering whether renaming the two objects would introduce too much clutter. Both Attachment objects are in separate package structures, so that should already help clarify the context of each object. (Also, in the code that actively uses the DTOs, it should theoretically be obvious which context the code is meant to use.)

I haven't found any cases in base code where two XML DTOs have the same name, but I did find some cases (such as with the Branch class) where the BO and an XML DTO have the same classname.

Perhaps we could discuss this further at the next bookclub meeting to get the team's input.